runtime/util/fltconv.c

```diff
@@ -440,4 +509,21 @@ fltconv_indexLeadingOne32(U_32 u32val)
 }
 
 
+static int 
+fltconv_indexLeadingOne64(U_64 u64val)
+{
+	int leading = 0;
+	
+	if (0 == u64val) {
+		return -1;
+	}
```

Comment:
This is dead code: the function is only called if `u64val != 0` (and the code below handles that case anyway).

Reply:
Removed.